mmdet/models/losses/iou_loss.py

```diff
@@ -237,6 +237,78 @@ def ciou_loss(pred, target, eps=1e-7):
     return loss
 
 
+@mmcv.jit(derivate=True, coderize=True)
```

Comment:
this decorator is unnecessary

Reply:
> this decorator is unnecessary

Okay, I will delete it in the next commit.